services/search.py

```diff
@@ -38,38 +51,40 @@ def search(query: str="",
                 },
                 "pipeline": [{
                     "$match": {
-                        "$expr": {"$eq": ["$_id", "$$c_id"]}
+                        "$expr": {
+                            "$eq": ["$_id", "$$c_id"]}
                     }
                 }],
                 "as": "companies"
             }
         },
-
-        {"$addFields": {"company": {"$arrayElemAt": ["$companies", 0]}}},
-
         {"$addFields": {"company": {"$arrayElemAt": ["$companies", 0]}}},
-
-        {"$addFields": {
-            "location.address": {
-                "postcode": "$location.postcode",
-                "address": "This is the address",
-                "city": "Test Town",
+        {
+            "$lookup": {
+                "from": "users",
+                "let": {
+                    "hm_ids": "$company.hire_managers_ids"
+                },
+                "pipeline": [{
+                    "$match": {
+                        "$expr": {
+                            "$in": ["$_id", "$$hm_ids"]}
+                    }
+                }],
+                "as": "company.hiring_managers"
             }
-        }},
-
-        {"$addFields": {"metadata.rate.pretty_print": "100£ per day"}},
-
-        {"$addFields": {"metadata.company_referent.full_name": "John Doe"}},
-        {"$addFields": {
-            "metadata.company_referent.phone_number": "+44 7873590126"}},
-
-        {"$addFields": {"duration.pretty_print": "3 months and 2 weeks"}},
-
-        {"$addFields": {"company.logo": "http://findastart.palmerminto.com/"
-                                        "wp-content/uploads/2018/06/"
-                                        "cropped-Find-a-Start-logo@2x.png"}},
-
-        {"$project": {"companies": 0}}
-    ]
-
-    return jobs.aggregate(pipeline)
+        },
+        {"$project": {
+            "companies": 0,
+            "company.hiring_managers.password": 0
+        }}
+    ])
+
+
+def _add_computed_fields_query(pipeline):
+    pipeline.append({
+        "$addFields": {
```

Comment:
The query now return real values in correct position
```
metadata.company_referent.full_name -> company.hiring_managers.first_name/last_name/title
metadata.company_referent.phone_numbe -> company. phone_number
company.logo -> Not required
```
Pick always the last `hiring_manage` from `company.hiring_managers`

"rate.pretty_print": "100£ per day"